src/plugins/discover/public/application/angular/discover.js

```diff
@@ -594,15 +605,27 @@ function discoverController($element, $route, $scope, $timeout, $window, Promise
     };
   };
 
+  function getDefaultColumns() {
```

Comment:
This is an attempt to remove `_source` from existing saved searches.

Reply:
might be better to do this with migrations.
loud thinking:  another option would be to don't use the new fields API for such cases, so source still would work, until the user reconfigures the table or our next major is here